netbsd/NetBSDProcessTable.c

```diff
@@ -164,7 +165,7 @@ static void NetBSDProcessTable_scanProcs(NetBSDProcessTable* this) {
       const struct kinfo_proc2* kproc = &kprocs[i];
 
       bool preExisting = false;
-      Process* proc = ProcessTable_getProcess(&this->super, kproc->p_pid, &preExisting, NetBSDProcess_new);
+      Process* proc = ProcessTable_getProcess(&npt->super, kproc->p_pid, &preExisting, NetBSDProcess_new);
```

Comment:
No need to use `&npt->super` if you have `super` as input argument already.
```suggestion
      Process* proc = ProcessTable_getProcess(super, kproc->p_pid, &preExisting, NetBSDProcess_new);
```

Reply:
Should be done in the updated commit.